### PR TITLE
Add envs and routes

### PR DESCRIPTION
**Pull request by benjefferies**
# Motivation and Context
* Sample service environmental variables were missing and have been added
* Added notifications for failed deployments
* Added map-routing for external domains

# What has changed
Add SAMPLE_PASSWORD, SAMPLE_URL, SAMPLE_USERNAME envs
Add on_failure for slack notifications
Add logic to map external domains

# How to test?
Check rh can be accessed in preprod
Pipeline should be green for latest and preprod (pipeline has been flown)
Check notification for failed builds

# Links
https://trello.com/c/SwkUMf9O/99-dev-task-7-ext-access-eq-social-survey-rh

### CODE REVIEW

File: pipelines/respondent-home-ui.yml [hunk 1]
```diff
@@ -72,6 +115,9 @@ jobs:
         SURVEY_URL: http://rm-survey-service-concourse-latest.apps.devtest.onsclofo.uk
         SURVEY_USERNAME: ((latest_security_user_name))
         SURVEY_PASSWORD: ((latest_security_user_password))
+        SAMPLE_PASSWORD: firm.wing.weather.legs.opinion.capital
```
Comment (jcox-dev) [inline pipelines/respondent-home-ui.yml]: Should these be `((latest_security_user_password))`?

**Review by jcox-dev — DISMISSED** [review]

File: pipelines/respondent-home-ui.yml (hunk 1 above)
Comment (benjefferies) [reply pipelines/respondent-home-ui.yml]: Good spot!

**Review by liamtoozer — APPROVED** [review]

**Review by tomphp — APPROVED** [review]
Looks good

File: pipelines/respondent-home-ui.yml (hunk 1 above)
Comment (tomphp) [reply pipelines/respondent-home-ui.yml]: Now that's hit github, it should be rotated!

Comment (benjefferies) [reply pipelines/respondent-home-ui.yml]: I'm not sure if it's that much of a problem as it's latest and no sensitive data lives there. I will check

**Pull request merged**